uptime_service_validation/coordinator/server.py

```diff
@@ -1,84 +1,267 @@
+                value=os.environ.get("AWS_KEYSPACE"),
+            ),
+            client.V1EnvVar(
+                name="AWS_S3_BUCKET",
+                value=os.environ.get("AWS_S3_BUCKET"),
+            ),
+            client.V1EnvVar(
+                name="CASSANDRA_HOST",
+                value=cassandra_ip,
+            ),
+            client.V1EnvVar(
+                name="CASSANDRA_PORT",
+                value=os.environ.get("CASSANDRA_PORT"),
+            ),
+            client.V1EnvVar(
+                name="CASSANDRA_USE_SSL",
+                value="1"
+            ),
+            client.V1EnvVar(
+                name="SSL_CERTFILE",
+                value="/root/.cassandra/sf-class2-root.crt",
+            ),
+            client.V1EnvVar(
+                name="CQLSH",
+                value=os.environ.get("CQLSH"),
+            ),
+            client.V1EnvVar(
+                name="AUTH_VOLUME_MOUNT_PATH",
+                value=os.environ.get("AUTH_VOLUME_MOUNT_PATH"),
+            ),
+            client.V1EnvVar(
+                name="NETWORK_NAME",
+                value=os.environ.get("NETWORK_NAME"),
+            ),
+            client.V1EnvVar(
+                name="START_TIMESTAMP",
+                value=datetime_formatter(mini_batch[0]),
+            ),
+            client.V1EnvVar(
+                name="END_TIMESTAMP",
+                value=datetime_formatter(mini_batch[1]),
+            ),
+        ]
+
+        # Entrypoint configmap name
+        entrypoint_configmap_name = (
+            f"{platform}-{network_name}-delegation-verify-coordinator-worker"
+        )
+
+        # Define the volumes
+        auth_volume = client.V1Volume(
+            name="auth-volume",
+            empty_dir=client.V1EmptyDirVolumeSource(),
+        )
+
+        entrypoint_volume = client.V1Volume(
+            name="entrypoint-volume",
+            config_map=client.V1ConfigMapVolumeSource(
+                name=entrypoint_configmap_name, default_mode=int("0777", 8)
+            ),  # 0777 permission in octal as int
+        )
+
+        # Define the volumeMounts
+        auth_volume_mount = client.V1VolumeMount(
+            name="auth-volume",
+            mount_path=os.environ.get("AUTH_VOLUME_MOUNT_PATH"),
+        )
+
+        entrypoint_volume_mount = client.V1VolumeMount(
+            name="entrypoint-volume",
+            mount_path="/bin/entrypoint",
+        )
+
+        # Define resources for app and init container
+        resource_requirements_container = client.V1ResourceRequirements(
+            limits={"cpu": worker_cpu_limit, "memory": worker_memory_limit},
+            requests={"cpu": worker_cpu_request, "memory": worker_memory_request},
+        )
+
+        # Define the container
+        container = client.V1Container(
+            name="delegation-verify",
+            image=f"{worker_image}:{worker_tag}",
+            command=["/bin/entrypoint/entrypoint-worker.sh"],
```

Comment:
Where does this script live? :thinking:

Reply:
In the kubernetes cluster. At the deployment of coordinator kubernetes creates a configmap with the contents of this script. The application than picks it up at lines 141-146 and mounts it in the container that it deploys (i.e. delegation verify)

Reply:
If it wouldn't exist in the cluster the application would fail. The other method was to give the coordinator permissions to deploy a configmap and deploy it themself, but then further management of said resource would have been needed. I guess it is more fail-safe but as long as we don't forget to deploy the configmap we should be good.

Reply:
I see. Perhaps some comment here about where those artifact live would be handy here.

Reply:
Added